lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py

```diff
@@ -93,7 +94,7 @@
 
 RETURN = '''
 state:
-    description: Current state of the availability set
+    description: Current state of the availability set.
     returned: always
     type: dict
```

Comment:
fill the contains, the options are mentioned in https://github.com/ansible/ansible/blob/254e9c62d411daed37c1f1b5c564a0db75e91a34/lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py#L115